apps/web/src/views/Swap/V3Swap/hooks/useSwapBestTrade.ts

```diff
@@ -58,7 +58,11 @@ export function useSwapBestOrder({ maxHops }: Options = {}) {
     stableSwap: stableSwapEnable,
     trackPerf: true,
     retry: 1,
-  })
+  }
+  const { fetchStatus, data, isStale, error, refetch } = useBestTradeFromApi(bestTradeOptions)
+  useBestTradeFromApiShadow(bestTradeOptions, 'quote-api-ori')
+  useBestTradeFromApiShadow(bestTradeOptions, 'quote-api-opt')
```

Comment:
let's enable for all chains. Previously the api is only enabled on arb and eth

Reply:
done